docs/src/whatsnew/3.1.rst

```diff
@@ -7,7 +7,7 @@ This document explains the changes made to Iris for this release
 (:doc:`View all changes <index>`.)
 
 
-.. dropdown:: :opticon:`report` Release Highlights
+.. dropdown:: :opticon:`report` v3.1.0 Release Highlights
```

Comment:
@pp-mo See [here](https://github.com/SciTools/iris/pull/4320/files#diff-8101c5764f46206f538e8ff80e0912f2d82ddd14d16c9872d0b73497540ee10bR9) for context.